pkg/reconciler/openshift/tektonhub/extension.go

```diff
@@ -45,9 +48,11 @@ import (
 )
 
 const (
-	hubprefix               string = "tekton-hub"
-	tektonHubAPIResourceKey string = "api"
-	tektonHubUiResourceKey  string = "ui"
+	hubprefix                  string = "tekton-hub"
+	tektonHubAPIResourceKey    string = "api"
+	tektonHubUiResourceKey     string = "ui"
+	CreatedByValue             string = "Tekton-Hub"
```

Comment:
```suggestion
	CreatedByValue             string = "TektonHub"
```
keeping similar to other component